app/src/androidTest/java/com/duckduckgo/app/browser/downloader/BlobConverterInjectorJsTest.kt

```diff
@@ -37,6 +38,7 @@ class BlobConverterInjectorJsTest {
 
     @UiThreadTest
     @Test
+    @SdkSuppress(minSdkVersion = 24)
```

Comment:
Currently, `spy` is not supported on API < 24
https://github.com/mockito/mockito/issues/2075#issuecomment-723507001